scripts/create-caddyfile.js

```diff
@@ -30,11 +30,19 @@ for (const serviceName in services) {
   if (serviceName === 'mongo' || serviceName === 'caddy') {
     continue;
   }
+
+  const exceptions = {
+    '25--5-clock': 'clock'
+  };
+  const subdomain = exceptions[serviceName]
+    ? exceptions[serviceName]
+    : serviceName;
```

Comment:
Probably a personal thing, but this is easier to look at.

```suggestion
  const subdomain = exceptions[serviceName] || serviceName;
```

Reply:
Since we've only got one exception, I don't think we need anything more complicated than
```
const subdomain = serviceName === '25--5-clock' ? 'clock' : serviceName;
```

Reply:
Thanks for your feedback @ShaunSHamilton and @ojeytonwilliams. I agree that both look better than what's currently there.

I probably over complicated things a bit, and hopefully we won't have other exceptions in the future. Let's go with Oliver's suggestion for now and fall back to Shaun's later if we need to.

Reply:
Sounds like a plan. I think Shaun's syntax is nice and clean, but let's keep things simple while we can!